package.json

```diff
@@ -78,6 +78,7 @@
     "diff": "~3.0.0",
     "file-saver": "^2.0.1",
     "gridster": "^0.5.6",
+    "history": "^4.7.2",
```

Comment:
do we need it as a top dependency or is it part of what react-router-dom requires?

Reply:
top